shared/redux/reducers/user.reducer.ts

```diff
@@ -25,22 +27,27 @@ export const authenticatedUserSlice = createSlice({
   reducers: {
     setUser: (state, action: PayloadAction<TTokenizedUser>) => {
       state.email = action.payload.email;
-      state.claimId = action.payload.claimId;
-      state.claim = action.payload.claim as EUserRole;
+      state.firstName = action.payload.firstName;
+      state.userType = action.payload.userType;
       state.userId = action.payload.id;
+      state.claimId = null;
```

Comment:
Either remove this from the state if unused as claim is already removed. Or rename this to userRoleId.

I'd remove it since I don't foresee a use case for it.